theatre/studio/src/uiComponents/Popover/usePopover.tsx

```diff
@@ -21,6 +21,10 @@ export default function usePopover(
     closeWhenPointerIsDistant?: boolean
     pointerDistanceThreshold?: number
     closeOnClickOutside?: boolean
+    minX?: number
+    maxX?: number
+    minY?: number
+    maxY?: number
```

Comment:
Suggestion: Might consider pulling these out into their own singular type alias which can be documented separately. e.g. `type AbsolutePlacementBoxConstraints = { minX?: number, ... }`, then that type can be reused in some of the interesting places.

Reply:
Pulled them out in `usePopover` and `tooltipWrapper`